### PR TITLE
Deren - Calendar app Backend

### PR DESCRIPTION
**Pull request by DerenB**
Created the backend setup and files for the Calendar App

- Created a Model for interacting with the mongo database
    - backend > src > models > `calendar_model.py`
- Created functions that use the Model for executing CRUD actions with the database
    - backend > src > dbfuncs > `calendar_db_funks.py`
- Created the routes for directing users to the calendar page and for calling the database functions
    - backend > src > paths > app > `calendar_routes.py`   
- Added the route/path import to `app.py`, following the same pattern as the other imports

### CODE REVIEW

File: backend/src/paths/app/calendar_routes.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# Import Database Functions
+from src.dbfuncs.calendar_db_funks import (
+    create_calendar_item,
+    read_one_calendar_item,
+    read_all_calendar_items,
+    update_calendar_item,
+    remove_calendar_item
+)
+
+# Create ToDo
```
Comment (srstone20) [inline backend/src/paths/app/calendar_routes.py:16]: Create calendar not todo

**Review by srstone20 — CHANGES_REQUESTED** [review]

File: backend/src/paths/app/calendar_routes.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DerenB) [reply backend/src/paths/app/calendar_routes.py]: Corrected this.

File: backend/src/paths/app/calendar_routes.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+@router.post("/calendar", tags=["calendar"], response_model=CalendarItem)
```
[the remark this answers is not in the document]
Comment (DerenB) [reply backend/src/paths/app/calendar_routes.py]: Corrected this.

File: backend/src/paths/app/calendar_routes.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+async def post_calendar_item(calendar_item:CalendarItem):
+    response = await create_calendar_item(calendar_item.dict())
+    if response:
+        return response
+    raise HTTPException(400, "Something went wrong, bad request") 
+
+
+# Read 1 ToDo
+@router.get("/calendar{title}", tags=["calendar"], response_model=CalendarItem)
```
[the remark this answers is not in the document]
Comment (DerenB) [reply backend/src/paths/app/calendar_routes.py]: Corrected this.

**Comment by DerenB** [comment]
I made those requested changes and added them to the PR. 👍

**Pull request merged**